target/linux/ramips/image/mt7621.mk

```diff
@@ -128,14 +128,15 @@ TARGET_DEVICES += alfa-network_quad-e4g
 
 define Device/ampedwireless_ally_common
   $(Device/dsa-migration)
+  $(Device/uimage-lzma-loader)
   DEVICE_VENDOR := Amped Wireless
   DEVICE_PACKAGES := kmod-mt7615e kmod-mt7615-firmware uboot-envtools
   IMAGE_SIZE := 32768k
   KERNEL_SIZE := 4096k
   BLOCKSIZE := 128k
   PAGESIZE := 2048
   UBINIZE_OPTS := -E 5
-  KERNEL_INITRAMFS := $(KERNEL_DTB) | uImage lzma -n 'flashable-initramfs' |\
+  KERNEL_INITRAMFS := $(KERNEL_DTB) | loader-kernel | lzma | uImage lzma |\
```

Comment:
Just to check, `KERNEL_DTB` already compresses the image, do you need to apply lzma twice?

And I guess `KERNEL` should also be changed for sysupgrade.bin, no?